ngraph/core/src/op/mod.cpp

```diff
@@ -12,58 +12,70 @@
 #include "ngraph/op/multiply.hpp"
 #include "ngraph/op/sign.hpp"
 #include "ngraph/op/subtract.hpp"
+#include "ngraph/runtime/reference/mod.hpp"
 
 using namespace std;
 using namespace ngraph;
 
-NGRAPH_SUPPRESS_DEPRECATED_START
-
-constexpr NodeTypeInfo op::v1::Mod::type_info;
-
-op::v1::Mod::Mod()
-    : FusedOp()
-    , m_auto_broadcast()
+namespace modop
 {
+    template <element::Type_t ET>
+    bool evaluate(const HostTensorPtr& arg0,
```

Comment:
@bszmelcz was addition of evaluate() for mod requested from someone? Originally ref impl was called from evaluates_map() so based on that I think we shouldn't add it. However, if it was explicitly requested then we need to remove it from evaluates_map.

Reply:
Oh yes yes, I forgot to remove it, since I already asked Gabri about it